CenterForOpenScience/osf.io
# [Feature] "Purge" Old TrashedFileNodes [ENG-???]

## Purpose
Reduce storage costs by purging inaccessible deleted files

## Changes
* ORM field, functions



## QA Notes
Largely Dev(Ops) QA. 
* Ensure "deleted" file page provides same metadata
* Ensure the same file in two locations, one active one deleted, is **not purged**
  * including when the deleted file was trashed before the active one was uploaded

## Deployment Notes
Bump Helm chart to `0.17.1`. 
Refer to staging configs for config changes.
Ensure SA existence, key conditions, and IAM configured. IAM should be configured in advance, as changes may take some time to propogate.

## Documentation
N/A

## Side Effects
None expected

## Ticket
None
